PelotonEppSdk/Models/Document.cs

```diff
@@ -18,13 +18,13 @@ public class Document
         /// <summary>
         /// Name that the document has when represented on a file system.
         /// </summary>
-        [Required, StringLength(128, ErrorMessage = "The " + nameof(Filename) + " field must be less than 128 characters long.")]
+        [Required, StringLength(128, ErrorMessage = "The " + nameof(Filename) + " field must be less than 128 characters in length.")]
         public string Filename { get; set; }
```

Comment:
Shouldn't this be "must be 128 or fewer characters in length" in be 1) Actually correct and 2) Consistent with the other similar validation messages.